gradle/init-scripts/build-scan.init.gradle.kts

```diff
@@ -0,0 +1,26 @@
+/*
+ * This is an init script for internal usage at Gradle Inc.
+ */
+initscript {
+    repositories {
+        maven {
+            setUrl("https://plugins.gradle.org/m2")
+        }
+    }
+    dependencies {
+        classpath("com.gradle:build-scan-plugin:1.12.1")
+    }
+}
+
+rootProject {
```

Comment:
:x: the check for the `disableScanPlugin` system property is missing in the Kotlin version of this script
see the Groovy version above